packages/compass-e2e-tests/helpers/commands/set-ace-value.ts

```diff
@@ -29,14 +29,36 @@ export async function setAceValue(
     return focused;
   });
 
+  if (process.env.COMPASS_E2E_DISABLE_CLIPBOARD_USAGE === 'true') {
+    const editorValue = await browser.execute(
+      (_selector, _value) => {
+        const editorNode =
+          document.querySelector(`${_selector} .ace_editor`) ||
+          document.querySelector(`${_selector}.ace_editor`);
+        if (!editorNode) {
+          throw new Error(
+            `Cannot find ace-editor container for selector ${_selector}`
+          );
+        }
+        const editor = (window as any).ace.edit(editorNode.id);
+        editor.setValue(_value);
```

Comment:
Any reason why we don't just always do this? ie. never bother with the clipboard.

Reply:
As we discussed on the sync, I'll make this helper always use the `ace.edit` way of changing input content